lib/node_modules/@stdlib/math/base/complex/divide/lib/internal-compreal.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+// MAIN //
+
+/**
+* Compute the real part of (re1 + i * im1) / (re2 + i * im2)
+*
+* @param {number} re1 - real component
+* @param {number} im1 - imaginary component
+* @param {number} re2 - real component
+* @param {number} im2 - imaginary component
+* @param {number} r - partial result from robustInternal
+* @param {number} t - partial result from robustInternal
```

Comment:
Having looked at the paper, `partial result` is fine. We don't need to give more meaning to these terms.